src/extensions/layout/breadthfirst.js

```diff
@@ -67,7 +67,7 @@ BreadthFirstLayout.prototype.run = function(){
 
         eles.bfs( {
           roots: unhandledNodes[0],
-          visit: function( node, edge, pNode, i, depth ){
+          visit: function( node ){
```

Comment:
This is an example of where it would be OK to remove the args, but I'd rather just remove the eslint rule instead of evaluating which should be removed and which ones not.  I'd rather address those when those particular parts of the code are being worked on and refactored.